packages/twenty-server/src/engine/metadata-modules/object-metadata/dtos/object-metadata.dto.ts

```diff
@@ -72,9 +72,9 @@ export class ObjectMetadataDTO {
   @Field()
   updatedAt: Date;
 
-  @Field({ nullable: true })
-  labelIdentifierFieldMetadataId?: string;
+  @Field(() => String, { nullable: true })
+  labelIdentifierFieldMetadataId?: string | null;
 
-  @Field({ nullable: true })
-  imageIdentifierFieldMetadataId?: string;
+  @Field(() => String, { nullable: true })
```

Comment:
This is needed because nestjs/graphql usually infers the type from the property bellow but since now it can be either a string or null it does not know what to do so you have to specify the return type in the Field decorator